torch/onnx/symbolic_opset11.py

```diff
@@ -80,6 +80,32 @@ def index_put(g, self, indices_list_value, values, accumulate=False):
         ]
         index = g.op("Concat", *indices_list, axis_i=-1)
     else:
+        # Replace index_put node with masked_scatter or masked_fill
+        # when inputs to the index_put node contains boolean inputs
+        # 
+        # (index_put -> masked_fill):
+        # graph(%0 : Float(2:4, 2:2, 2:1, requires_grad=0, device=cpu)):
+        # %mask.1 : Float(2:4, 2:2, 2:1, requires_grad=0, device=cpu)
+        # %22 : Tensor?[] = prim::ListConstruct(%21)
+        # %23 : Float(requires_grad=0, device=cpu) = prim::Constant[value={1}]()
+        # %24 : bool = prim::Constant[value=0]()
+        # %mask : Float(2:4, 2:2, 2:1) = aten::index_put_(%mask.1, %22, %23, %24)
+        # 
+        # after the change
+        # graph(%0 : Float(2:4, 2:2, 2:1, requires_grad=0, device=cpu)):
+        # %46 : Float(requires_grad=0, device=cpu) = prim::Constant[value={5}]()
+        # %mask.1 : Float(2:4, 2:2, 2:1, requires_grad=0, device=cpu) =
+        # %23 : Float(requires_grad=0, device=cpu) = prim::Constant[value={1}]()
+        # %24 : bool = prim::Constant[value=0]()
+        # %49 : Tensor = aten::masked_fill(%mask.1, %21, %23)
```

Comment:
Please update the IR graph above since it is no longer the same now that this is at onnx conversion.